src/typescript.js

```diff
@@ -69,7 +67,6 @@ module.exports = {
       rules: {
         '@typescript-eslint/no-unsafe-call': 'off',
         '@typescript-eslint/no-unsafe-member-access': 'off',
-        'unicorn/no-array-for-each': 'warn',
```

Comment:
I changed it because was conflicting with [no-restricted-syntax](https://eslint.org/docs/rules/no-restricted-syntax).

I am testing everything before creating the PR.

But apparently, I made some mistake, because I re-checked now and it is working fine.

So, I have reverted this change ✅